docs/ape17.rst

```diff
@@ -0,0 +1,537 @@
+generated in :ref:`Step 0 <step0>`, and you can then see if any previous customizations you had
+made need to be copied over. This file shows how one can configure Travis to use
+tox, optionally using conda via ci-helpers to set up Python on MacOS X and
+Windows.
+
+Step 14 - Update ReadTheDocs configuration
+------------------------------------------
+
+With the set-up described in this migration guide, you should be able to
+simplify the configuration for ReadTheDocs. This can be done via a
+``readthedocs.yml`` or ``.readthedocs.yml`` file (the latter is recommended).
+You should be able to copy over the ``.readthedocs.yml`` file created in :ref:`Step 0 <step0>`.
+With this updated file, you should now be able to remove any pip requirements
+file or conda yml file that were previously used by ``readthedocs.yml``.
+
+Step 15 - Coverage configuration
+--------------------------------
+
+Preivously, astropy-helpers expected the coverage configuration to
+be located in ``my_package/tests/coveragerc``. This is now no longer
+necessary, so you can now define the coverage configuration inside
+the ``setup.cfg`` file, which should help reduce the number of files
+to keep track of. Add the following to the bottom of your ``setup.cfg``::
+
+    [coverage:run]
+    omit =
+        my_package/_{{ cookiecutter._parent_project }}_init*
+        my_package/conftest.py
+        my_package/*setup_package*
+        my_package/tests/*
+        my_package/*/tests/*
+        my_package/extern/*
+        my_package/version*
+        */my_package/_{{ cookiecutter._parent_project }}_init*
+        */my_package/conftest.py
+        */my_package/*setup_package*
+        */my_package/tests/*
+        */my_package/*/tests/*
+        */my_package/extern/*
+        */my_package/version*
+
+    [coverage:report]
+    exclude_lines =
+        # Have to re-enable the standard pragma
+        pragma: no cover
+        # Don't complain about packages we have installed
+        except ImportError
+        # Don't complain if tests don't hit assertions
+        raise AssertionError
+        raise NotImplementedError
+        # Don't complain about script hooks
+        def main\(.*\):
+        # Ignore branches that don't pertain to this version of Python
+        pragma: py{ignore_python_version}
+        # Don't complain about IPython completion helper
+        def _ipython_key_completions_
+
+Make sure to replace ``my_package`` by your module name. If you had any
+customizations in ``coveragerc`` you can include them here, but otherwise the
+above should be sufficient.
+
+Step 16 - conftest.py file updates
+----------------------------------
+
+For the header in your test runs to be correct with the latest versions of
+astropy, you will need to make sure that you update your ``conftest.py`` file as
+described in the `pytest-astropy-header instructions
+<https://github.com/astropy/pytest-astropy-header#migrating-from-the-astropy-header-plugin-to-pytest-astropy>`_.
```

Comment:
It would be nice to close https://github.com/astropy/pytest-astropy-header/issues/14 before the migration guide is distributed widely. I found that error particularly confusing.

Reply:
See https://github.com/astropy/pytest-astropy-header/pull/16

Reply:
https://github.com/astropy/pytest-astropy-header/pull/16 has been merged, so resolving this